### PR TITLE
Add yml for the new service directory spring-boms

### PR DESCRIPTION
**Pull request by moarychan**
Adding yml files to setup pipelines, it's used to test the service directory under the spring-boms.

### CODE REVIEW

File: sdk/spring-boms/spring-cloud-azure-dependencies-3/pom.xml [hunk 1]
```diff
@@ -0,0 +1,312 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.azure.spring</groupId>
+  <artifactId>spring-cloud-azure-dependencies</artifactId>
+  <version>4.4.0-beta.1</version> <!-- {x-version-update;com.azure.spring:spring-cloud-azure-dependencies;current} -->
```
Comment (saragluna) [inline sdk/spring-boms/spring-cloud-azure-dependencies-3/pom.xml:9]: version not correct

**Review by saragluna — CHANGES_REQUESTED** [review]

File: sdk/spring-boms/spring-cloud-azure-dependencies-3/pom.xml (hunk 1 above)
Comment (moarychan) [reply sdk/spring-boms/spring-cloud-azure-dependencies-3/pom.xml:9]: all the versions will be updated in the test pr.

Comment (saragluna) [reply sdk/spring-boms/spring-cloud-azure-dependencies-3/pom.xml:9]: What did you mean by "all the versions will be updated in the test pr"?

Comment (moarychan) [reply sdk/spring-boms/spring-cloud-azure-dependencies-3/pom.xml:9]: It means the versions within the spring-cloud-azure-dependencies-3 will be updated in this [PR](https://github.com/Azure/azure-sdk-for-java/pull/30067).

**Comment by moarychan** [comment]
Close it because we decide to use a multi-branch to support different Spring Boot versions.

**Pull request closed without merging**